From 12d45d250d08e96e889e38e77273c3ef73e6fc97 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 25 Nov 2019 15:07:35 -0800
Subject: [PATCH] pass LDFLAGS to link step

This helps to use OE specific linker flags and fixes

do_package_qa: QA Issue: No GNU_HASH in the ELF binary

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8a605fd..d1c9637 100644
--- a/Makefile
+++ b/Makefile
@@ -20,7 +20,7 @@ ${LIBUCONTEXT_STATIC_NAME}: ${LIBUCONTEXT_OBJ}
 
 ${LIBUCONTEXT_NAME}: ${LIBUCONTEXT_OBJ}
 	$(CC) -o ${LIBUCONTEXT_NAME} -Wl,-soname,${LIBUCONTEXT_SONAME} \
-		-shared ${LIBUCONTEXT_OBJ}
+		-shared ${LIBUCONTEXT_OBJ} ${LDFLAGS}
 
 ${LIBUCONTEXT_SONAME}: ${LIBUCONTEXT_NAME}
 	ln -sf ${LIBUCONTEXT_NAME} ${LIBUCONTEXT_SONAME}
-- 
2.24.0

